examples/js/MarchingCubes.js

```diff
@@ -170,6 +187,31 @@ THREE.MarchingCubes = function ( resolution, material, enableUvs, enableColors )
 			field6 = scope.field[ qyz ],
 			field7 = scope.field[ q1yz ];
 
+		var c_field0 = scope.palette
+				.slice( q * 4, q * 4 + 3 )
+				.map( v => v / scope.palette[ q * 4 + 3 ] ), // slicing (r, g, b) part
```

Comment:
I bet these `.slice()` and `.map()` calls have a lot to do with the performance regression.

Reply:
This part was critical to enhancing performance(thank you). I deleted this part and pass start index of color array to `VIntX`, `VIntY` and `VIntZ`.